.github/workflows/hq-smoke-tests.yml

```diff
@@ -63,77 +77,59 @@ jobs:
-          priority: normal
+            const {JOB_STATUS, NOW, CC_ENV, GITHUB_HEAD_REF} = process.env
+            const prefix = `[${CC_ENV}] Smoke Tests - ${JOB_STATUS.toUpperCase()} - Run #${context.runNumber}`
+            const suffix = `at ${NOW}`
 
-      - name: Send Failure Email on Push or Pull
-        uses: dawidd6/action-send-mail@v3
-        if: ${{ failure() && github.event_name != 'repository_dispatch' }}
-        with:
-          server_address: smtp.gmail.com
-          server_port: 465
-          username: ${{secrets.DIMAGIQA_MAIL_USERNAME}}
-          password: ${{secrets.DIMAGIQA_MAIL_PASSWORD}}
-          subject: FAIL - Smoke Tests[#${{github.run_number}}] for ${{ matrix.environment }} on branch "${{ github.head_ref }}", ${{env.NOW}}
-          to: qa-automation@dimagi.com
-          from: <${{secrets.DIMAGIQA_MAIL_USERNAME}}>
-          html_body: file:///${{ github.workspace }}/HQSmokeTests/email_fail.html
-          attachments: ${{ github.workspace }}/report_${{ matrix.environment }}.html
-          priority: normal
+            let subject = `${prefix} on "deploy_sucess" ${suffix}`
+            if (context.eventName !== "repository_dispatch") {
+              subject = `${prefix} on branch "${GITHUB_HEAD_REF}" ${suffix}`
+            }
+
+            let bodyFile = './HQSmokeTests/email_pass.md'
+            if (JOB_STATUS !== 'success') {
+              bodyFile = './HQSmokeTests/email_fail.md'
+            }
+
+            let actionRunLink = context.payload.repository.html_url + `/actions/runs/${context.runId}`
+            let bodyContent = await fs.readFile(bodyFile, 'utf8')
+            bodyContent = bodyContent.replace(/{{actionRunLink}}/g, actionRunLink)
+                .replace(/{{runNumber}}/g, context.runNumber)
 
-      - name: Send Pass Email on Push or Pull
+            return {
+              "subject": subject,
+              "body": bodyContent
+            }
+
+      - name: Send Result Email
+        if: always()
         uses: dawidd6/action-send-mail@v3
-        if: ${{ success() && github.event_name != 'repository_dispatch' }}
         with:
           server_address: smtp.gmail.com
           server_port: 465
           username: ${{secrets.DIMAGIQA_MAIL_USERNAME}}
           password: ${{secrets.DIMAGIQA_MAIL_PASSWORD}}
-          subject: PASS - Smoke Tests[#${{github.run_number}}] for ${{ matrix.environment }} on branch "${{ github.head_ref }}", ${{env.NOW}}
-          to: qa-automation@dimagi.com
+          subject: ${{ fromJSON(steps.configure_email.outputs.result).subject }}
+          to: qa@dimagi.com, sameena.shaik@fissionlabs.com
           from: <${{secrets.DIMAGIQA_MAIL_USERNAME}}>
-          html_body: file:///${{ github.workspace }}/HQSmokeTests/email_pass.html
-          priority: normal
+          html_body:  ${{ fromJSON(steps.configure_email.outputs.result).body }}
+          convert_markdown: true
 
```

Comment:
do we add the attachments here or set it in the `configure_email` and call is fromJSON like the subject and body?

Reply:
good point, I think the version of the step I coped was a failure so there were no attachments. I'll add that back.